### PR TITLE
17.0 estbaivendorbill jco

### PR DESCRIPTION
**Pull request by jco-odoo**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_es_edi_tbai/models/account_move.py [hunk 1]
```diff
@@ -80,7 +81,8 @@ class AccountMove(models.Model):
     @api.depends('move_type', 'company_id')
     def _compute_l10n_es_tbai_is_required(self):
         for move in self:
-            move.l10n_es_tbai_is_required = move.is_sale_document() \
+            move.l10n_es_tbai_is_required = (move.is_sale_document() or move.is_purchase_document() and move.company_id.l10n_es_tbai_tax_agency == 'bizkaia'
```
Comment (qdp-odoo) [inline addons/l10n_es_edi_tbai/models/account_move.py:84]: so vouchers (in_receipts, out_receipts) are not to be included? It doesn't seem very consistent: previously we were checking if move_type.startswith('xxx'), or move_type in (...)

Probably better to choose one and stick to it for the whole module, imo

**Review by qdp-odoo — COMMENTED** [review]
made a review :)

**Comment by qdp-odoo** [comment]
@jco-odoo Are we still waiting for something or can I r+ this PR?

**Comment by jco-odoo** [comment]
@qdp-odoo I have just checked one last thing and it is ok.  Please do.

**Comment by qdp-odoo** [comment]
@robodoo r+

**Pull request closed without merging**